src/strangerThings.js

```diff
@@ -11,7 +10,19 @@ function sleep(ms) {
 TODO:
 This function should return a list of all the distinct characters in UPPERCASE that have been typed in the textbox with the id "message"
 */
-function getCharacters() {}
+function getCharacters() {
+	let msg = document.getElementById("message").value;
+	let str = [];
+	console.log(msg.length);
```

Comment:
Remove unnecessary console.log from everywhere in your code before submissions.

Reply:
Oh sorry bhaiya

Reply:
Nothing to be sorry about, lol. It's just a common practice.